js/calculation.js

```diff
@@ -0,0 +1,375 @@
+        }
+    }
+        
+        
+        
+        //This function shows the calculate button to add a layer of safety//
+    function safety() {
+        var disclaimer = document.getElementById('calc');
+        
+         if (checkbox.checked) {
+            disclaimer.style.display = 'block';
+        } else {
+            disclaimer.style.display = 'none';
+        }
+    }
+
+        //Used to scroll form down//
+    function scrollWin() {
+     window.scrollBy(0, 300);
+        }
+        
+    // this function displays the dosing table and calualtions when the "show dosing rationale" button is clicked//
+ 
+    function explain() {
+        var explainRationale = document.getElementById('explainDosing');
+        
+        if (checkbox.checked) {
+            explainRationale.style.display = 'block';
+        } else {
+            explainRationale.style.display = 'none';
+        }
+    }
+        
+    //This funtion explains the calculation to the user//
+    function displayReasoning() {
+        var showFormulas = document.getElementById("showFormulas");
+        var gender;
+        var weightStatus;
+        var CrClCalc;
+        var bmiFormula = "weight/(height in m)²"
+
+        if (m.checked) {
+            gender = "male"
+        } else if (f.checked) {
+            gender = "female"
+        }
+        
+        if (bmi >=25) {
+            weightStatus = "overweight, therefore their ideal body weight has been calculated as " + IBW + "kg" + " and used in the CrCl formula"
+        } else if (bmi <18.5) {
+            weightStatus = "underweight, therefore their creatinine clearence has been adjusted by multiplying the result by 0.69."
+        } else {
+            weightStatus ="of normal weight, therefore no adjustment to creatinine clearence is required."
+        }
+        
+        if (bmi >=25 && m.checked){
+                CrClCalc = "((140-age)* IBW * 1.23)/creatinine";
+            }else if (bmi >=25 && f.checked){
+                CrClCalc = "((140-age)* IBW * 1.04)/creatinine";
+            }else if  (bmi <18.5 && m.checked){
+                CrClCalc = "(((140-age) * weight * 1.23)/creatinine) * 0.69";
+            }else if (bmi <18.5 && f.checked){
+                CrClCalc = "(((140-age) * weight * 1.04)/creatinine) * 0.69";
+            }else if  (bmi >18.5 && bmi <25 && m.checked){
+                CrClCalc = "((140-age) * weight * 1.23)/creatinine";
+            }else if (bmi >18.5 && bmi <25 && f.checked){
+                CrClCalc = "((140-age) * weight * 1.04)/creatinine";
+        }
+        
+        if (CRRT.checked) {
+                showFormulas.innerHTML = "The patient is a " + "<b>" + a.value + " year old " + gender + "." + "</b>" + " Their weight is " + "<b>" + w.value + "kg." + "</b>";
+            } else{
+        showFormulas.innerHTML = "The patient is a " + "<b>" + a.value + " year old " + gender + "." + "</b>" + " Their weight is " + "<b>" + w.value + "kg." + "</b>"
```

Comment:
template literals will make this look much better
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals